bodo/tests/test_iceberg/test_stream_iceberg_write.py

```diff
@@ -207,14 +207,7 @@ def check_table_comment(
             )
 
 
-@pytest.mark.timeout(1000)
-@pytest.mark.parametrize("column_comments", ["test_col_comments", None])
-@pytest.mark.parametrize("table_comments", ["test_tbl_comments", "", None])
-@pytest.mark.parametrize("table_properties", [True, False])
 def test_iceberg_write_basic(
```

Comment:
So originally, this test would test Iceberg stream write on all types with all combinations of column comments, table comments, and table properties. That's completely unnecessary since comments and properties are not affected by the dataframe type. So I split it into two separate tests. That made these tests 15-20 minutes faster